src/components/Table/Table.tsx

```diff
@@ -0,0 +1,228 @@
+      }
+
+      if (father) {
+        newPerson.father = father;
+      }
+
+      return newPerson;
+    }),
+    [people],
+  );
+
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {isLoading && <Loader />}
+
+        {isError
+          ? (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              {errMsg}
+            </p>
+          )
+          : (infoMsg.length > 0 && (
+            <p data-cy="noPeopleMessage">
+              {infoMsg}
+            </p>
+          ))}
+
+        {preparedPeople.length > 0
+          && (
+            <table
+              data-cy="peopleTable"
+              className="table is-striped is-hoverable is-narrow is-fullwidth"
+            >
+              <thead>
+                <tr>
+                  <th>Name</th>
+                  <th>Sex</th>
+                  <th>Born</th>
+                  <th>Died</th>
+                  <th>Mother</th>
+                  <th>Father</th>
+                </tr>
+              </thead>
+
+              <tbody>
+                {preparedPeople.map(person => (
+                  <tr
+                    data-cy="person"
+                    key={person.slug}
+                    className={classNames(
+                      { 'has-background-warning': person.slug === personSlug },
+                    )}
+                  >
+                    <td>
+                      <Link
+                        className={classNames(
+                          { 'has-text-danger': person.sex === 'f' },
+                        )}
+                        to={`../${person.slug}`}
+                      >
+                        {person.name}
+                      </Link>
+                    </td>
+
+                    <td>{person.sex}</td>
+                    <td>{person.born}</td>
+                    <td>{person.died}</td>
+                    <td>
+                      {person.mother
+                        ? (
+                          <Link
+                            to={`../${person.mother.slug}`}
+                            className="has-text-danger"
+                          >
+                            {person.mother.name}
+                          </Link>
+                        )
+                        : person.motherName}
+                    </td>
+                    <td>
+                      {person.father
+                        ? (
+                          <Link to={`../${person.father.slug}`}>
+                            {person.father.name}
+                          </Link>
+                        )
+                        : person.fatherName }
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            </table>
+          )}
+
+        {/* <tr data-cy="person">
+          <td>
+            <a href="#/people/philibert-haverbeke-1907">
+              Philibert Haverbeke
+            </a>
+          </td>
```

Comment:
how come 😢

Reply:
realy shit)), i forgot about this commented parts of code))) thx